src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/dto/cart/CartProject.java

```diff
@@ -0,0 +1,35 @@
+package ca.corefacility.bioinformatics.irida.ria.web.ajax.dto.cart;
+
+import java.util.List;
+
+import ca.corefacility.bioinformatics.irida.ria.web.ajax.dto.CartSample;
+
+/**
+ * Model for UI to represent a project and its sample within the cart.
+ */
+public class CartProject {
```

Comment:
As this is a model for the UI rename it to `CartProjectModel`

Reply:
Updated in 355003282dbbbbf2cc92247eb2a8b80b5b121a5f